frontend/src/types.ts

```diff
@@ -1331,7 +1331,7 @@ export interface CurrentBillCycleType {
 }
 
 export type BillingV2FeatureType = {
-    key: AvailableFeatureUnion
+    key: AvailableFeature
```

Comment:
I think having this as a union was important for the json billing test fixtures. Those jsons have feature keys coded as strings and using the union hack keeps typescript happy. I suppose the other way around this might be to change all the keys in the json to proper enum values, but that felt like too much work at the time I submitted the jsons.